### PR TITLE
Spelling and DOI fixes

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

These are some fixes to DOIs that were formatted improperly and appearing as [???] in the build, as well as a word I forgot to add to the custom dictionary at merge.

### Related issues

#486 is failing to build and most of the error messages are about DOIs, so hoping this will help simplify debugging at least!

### CODE REVIEW

File: content/02.introduction.md [hunk 1]
```diff
@@ -75,15 +75,15 @@ The current pandemic of COVID-19, caused by the severe acute respiratory syndrom
...
 For SARS-CoV-2 it is still being investigated whether, in addition to being spread by people who show symptoms,  the virus can be transmitted by people who do not show symptoms [@url:https://www.cdc.gov/coronavirus/2019-ncov/prevent-getting-sick/how-covid-spreads.html], such as during the pre-symptomatic stage of infection or in people with asymptomatic infections (a presentation which sometimes occurs in other infectious diseases but is still being investigated for SARS-CoV-2).
 
 #### Cell Entry and Replication
 
 Coronavirus virions are spherical with diameters ranging from 100 to 160 nanometers (nm).
-The virion is made up of a lipid envelope in which peplomers of 2-3 spike (S) glycoproteins are anchored, creating a distinctive "crown" shape for which the family of viruses was named [@isbn:0781760607; @doi:10.1128/JVI.80.3.1302–1310.2006].
+The virion is made up of a lipid envelope in which peplomers of 2-3 spike (S) glycoproteins are anchored, creating a distinctive "crown" shape for which the family of viruses was named [@isbn:978-0781760607; @doi:10.1128/JVI.80.3.1302–1310.2006].
```
Comment (rando2) [inline content/02.introduction.md:86]: Not sure about this one, trying the ISBN-13

Comment (agitter) [reply content/02.introduction.md:86]: `isbn:0781760607` and `isbn:978-0781760607` both work in my local testing

**Comment by rando2** [comment]
Thank you @agitter! Looking at the logs, I actually think it might be the last changes pushed to metadata.yaml (prior to your corrections) in #488 that are causing build issues -- is that possible?

**Comment by agitter** [comment]
I haven't looked closely at all the build logs, but my best guess is that the metadata build error is isolated to #488.  All the other failures are caused by #497.  If #497 is blocking a lot of pull request merges, I could disable the Python script temporarily until we resolve the underlying issue.

**Comment by rando2** [comment]
@agitter OK that is reassuring! I'm going to go ahead and merge PRs where I'm able to see the manuscript built OK through appveyor, if that sounds OK to you.

**Pull request merged**